zng/ztests/enum-err.yaml

```
@@ -0,0 +1,9 @@
+zql: "put s=e:string"
```

```suggestion
zql: put s=e:string
```
Most of the new ztests also quote the zql field's value unnecessarily.

---

I changed this to a script test now that it looks for a stderr result per @henridf's feedback.